test/linter/test-versions.js

```diff
@@ -101,7 +101,7 @@ function checkVersions(supportData, relPath, logger) {
  * @param {string} filename
  */
 function testVersions(filename) {
-  const relativePath = path.relative(path.resolve(__dirname, '..'), filename);
+  const relativePath = path.relative(path.resolve(__dirname, '..', '..'), filename);
```

Comment:
`relativePath` is never used? Maybe I'm overseeing something or this is a new (but separate) issue.

Reply:
Yeah, it’s not being used anymore.